files/ib_perftest.yml

```diff
@@ -0,0 +1,133 @@
+                         hpc-sig-devel:
+                                 - job-build
+                                 - job-read
+         parameters:
+                 - node:
+                         name: cluster
+                         allowed-slaves:
+                                 - d05ohpc
+                                 - qdcohpc
+                         allowed-multiselect: False
+                         description: 'Which cluster to use'
+                 - string:
+                         name: automation_branch
+                         default: 'production'
+                         description: 'Branch name of the ib_perftest recipe'
+                 - string:
+                         name: client_branch
+                         default: 'production'
+                         description: 'Branch name of the MrP client to use'
+         builders:
+                 - shell: |
+                        #!/bin/bash
+                        set -ex
+                        if [ -d ${WORKSPACE}/mr-provisioner-client ]; then
+                            rm -rf ${WORKSPACE}/mr-provisioner-client
+                        fi
+                        git clone -b ${client_branch} https://github.com/Linaro/mr-provisioner-client.git ${WORKSPACE}/mr-provisioner-client
+
+                        mr_provisioner_url='http://10.40.0.11:5000'
+                        mr_provisioner_token=$(cat "/home/${NODE_NAME}/mrp_token")
+
+                        if [ ${cluster} == 'd05ohpc' ]; then
+                                host_machine='d05ohpc'
+                                remote_machine='d0301'
+                        fi
+
+                        if [ ${cluster} == 'qdcohpc' ]; then
+                                host_machine='qdcohpc'
+                                remote_machine='qdc01'
+                        fi
+
+                        host_machine_ip=$( ${WORKSPACE}/mr-provisioner-client/mrp_client.py getip ${host_machine} --mrp-url=${mr_provisioner_url} --mrp-token=${mr_provisioner_token})
+                        remote_machine_ip=$( ${WORKSPACE}/mr-provisioner-client/mrp_client.py getip ${remote_machine} --mrp-url=${mr_provisioner_url} --mrp-token=${mr_provisioner_token})
+
+                        # Get the XXX in 10.40.24.XXX to make 192.168.100.XXX
+                        IFS="." read -ra REMOTE <<< "$remote_machine_ip"
+                        remote_ib_ip="192.168.100.${REMOTE[-1]}"
+                        IFS="." read -ra HOST <<< "$host_machine_ip"
+                        host_ib_ip="192.168.100.${HOST[-1]}"
+
+                        # We'll be using the same port number for all machines
+                        host_ib_devnb='1'
+                        remote_ib_devnb='1'
+
+                        # Prepare the 'tests' dictionary
+                        tests = ''
+                        if [ ${do_perftest} == 'True' ]; then
+                                tests+="- name: bw\n  threshold: '45.0'\n"
```

Comment:
thresholds need to be in the python/ansible code, not Jenkins. This job should remain unchanged (ideally) for the rest of its life. You can make it accept parameters, if you want, but the defaults should be in there.